### PR TITLE
Added Task_3 script

### PR DESCRIPTION
**Pull request by sggts04**
__21__ 

#### Short description of what this resolves:
Adds task-3 script for sending mail

### CODE REVIEW

File: Submissions/sggts04/Task_3/sendmail.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import smtplib
+
+smtp_obj = smtplib.SMTP('smtp.gmail.com', 587)
+smtp_obj.ehlo()
+smtp_obj.starttls()
+
+email = ''
+passwd = ''
+
+smtp_obj.login(email, passwd)
```
Comment (Lokesh2703) [inline Submissions/sggts04/Task_3/sendmail.py:12]: How can you log in with empty mail ID and password

Comment (sggts04) [reply Submissions/sggts04/Task_3/sendmail.py:12]: The user will enter their email and password. I used mine and then removed when committing.

**Comment by Lokesh2703** [comment]
Then write code for taking the input from the user

**Comment by sggts04** [comment]
@Lokesh2703 done

**Comment by Lokesh2703** [comment]
Quit the server after sending mail.

**Comment by sggts04** [comment]
done

**Pull request merged**